### PR TITLE
feat/register

### PR DESCRIPTION
**Pull request by mikser124**
please review

### CODE REVIEW

File: src/Register.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return password;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
```
Comment (adriangazdowicz) [inline src/Register.java:37]: delete this

**Review by adriangazdowicz — COMMENTED** [review]
delete unnecessary setters

File: src/Register.java (hunk 1 above)
Comment (mikser124) [reply src/Register.java:37]: done, check

Comment (adriangazdowicz) [reply src/Register.java:37]: good

**Review by pinkcone — APPROVED** [review]
ok

**Pull request merged**